### DISCUSSION TITLE
Blank Page on port port 8009

### DISCUSSION THREAD

**Discussion by buzzzo** [hoophq/hoop#543] [General]
I've installed hoop as per documentation on a rocky linux 8 via docker.
All containers appear to be run buit whet i try to connect via browser on http://x.y.z.j:8009 i receive a blank pag
It's not a connection refused error, the page loads (i've the text "hoop" in the chrome's tab ) BUT it's blank.

How can i troubleshoot the issue ?

Thx

**Comment by sandromello** [comment]
The Webapp uses the value from the environment variable `API_URL`, it's important that this value is set to your public dns
By default it uses `http://127.0.0.1:8009`.

Did you followed [this article](https://hoop.dev/docs/getting-started/installation/docker-compose)?

**Comment by buzzzo** [comment]
You may need to improve the docs, it lacks some info (as for example how to manage ssl etc etc).
Thx now i'm able to start the web ui.
